### ISSUE TITLE
flask-classy continuing development and maintenance

### ISSUE THREAD

**Issue by hoatle** [apiguy/flask-classy#80]
Hello guys,

flask-classy is awesome, however, I don't know why @apiguy haven't responded for a while with PR and issues.

So I forked the project and add more features, supports for it here at: https://github.com/teracyhq/flask-classful

and the updated documentation here at: http://flask-classful.teracy.org/

One thing remains is that I don't have flask-classy pypi credentials to updating it on pypi.

I hope that @apiguy will see this and provide me the credentials to update on pypi.

Or maybe I'll change the flask-classy to something like: flask-class to publish on pypi.

Let's see how we could come up with this project.

**Comment by OOPMan** [comment]
@hoatle Hi, nice to see you're taking over maintenance of this. With regards to the PRs against this repo, have any or all of them been merged in to your fork?

**Comment by hoatle** [comment]
@OOPMan yes, I think I merged some and added some more updates, please checkout the commits here at: https://github.com/teracyhq/flask-classy/commits/develop

**Comment by OOPMan** [comment]
@hoatle Great. Just wanted to check as I will probably be adding your fork to a project I'm working on...

**Comment by hoatle** [comment]
@OOPMan yes, and you should access the update the latest docs at: http://flask-classy.teracy.org/ (on master branch) and http://flask-classy.teracy.org/develop/ on develop branch

**Comment by bivald** [comment]
I've pinged @apiguy on Twitter as well regarding this. I'll also use @hoatle's fork in my projects from now on.

**Comment by lucidfrontier45** [comment]
@hoatle 
Thank you for inheriting the project.
I have a question about PyPI registration. Do you have any intention to register your fork as another package name? I don'k now if you can use @apiguy 's PyPI account.

I think it is more convenient if the package can be found on PyPI.

**Comment by bivald** [comment]
@lucidfrontier45 ,

@hoatle would need access to @apiguy to publish under the same name (even though doing so would absolutely be the best) or start a claiming process. It would be better if @hoatle could be invited to this repo of course.

@hoatle You can open a support request for Pypi to claim ownership of a abandoned project by visiting http://sourceforge.net/p/pypi/support-requests/ and creating a new support request. Looking at https://mail.python.org/pipermail/distutils-sig/2014-September/024841.html they will they try to contact @apiguy and if they fail they might grant ownership transfer

That said, it might be simpler to create a new fork named flask-classier (or something) :8ball:

**Comment by svenstaro** [comment]
I suggest you just rename your work and that's that. Maybe call it flask-classful for maximum confusion in the ecosystem.

**Comment by hoatle** [comment]
thank you guys, let me try to claim ownership of a abandoned project first.

**Comment by hoatle** [comment]
issue created: https://sourceforge.net/p/pypi/support-requests/591/

**Comment by hoatle** [comment]
FYI, pypi ownership transfer issue was accepted https://sourceforge.net/p/pypi/support-requests/591/

**Comment by hoatle** [comment]
FYI, v0.8.0 has just been released https://github.com/teracyhq/flask-classy/blob/master/CHANGELOG.md#2016-02-16-hoat-le-hoatleteracycom

**Comment by bivald** [comment]
:+1:

**Comment by iartarisi** [comment]
To clarify, it seems that you have not yet been granted pypi access? And the "0.8.0 released" does not mean "released on pypi", but rather "tagged in github"?

**Comment by hoatle** [comment]
@mapleoin yes, I meant github tagged, it will be pypi released when I got pypi ownership transfer.

**Comment by hoatle** [comment]
https://sourceforge.net/p/pypi/support-requests/591/ closed with response:

`The owner has responded that this is not an abandoned project.`

**Comment by bivald** [comment]
Perhaps you @apiguy could add @hoatle as maintainer of this repo then?

**Comment by EvaSDK** [comment]
Did the pypi guys ignore the absence of activity on this repo ? I fail to see how this is not abandoned considering the lack of commits for over a year and a half.

**Comment by OOPMan** [comment]
@hoatle Time to rename your fork?

**Comment by OOPMan** [comment]
And release it under the new name?

**Comment by svenstaro** [comment]
I suggest `flask-classful` :P

**Comment by hoatle** [comment]
I guess so, what's about `flask-classify` as a new name? :D

**Comment by bivald** [comment]
I like flask-classful :+1:

**Comment by OOPMan** [comment]
`flask-class-act` :-)

**Comment by hoatle** [comment]
thank you guys, let's go with `flask-classful` then, I'll update on this issue when it's done.

**Comment by hoatle** [comment]
hello guys, let's make it clear with this change. Please let me know if there is any problem with the changes here: https://github.com/teracyhq/flask-classful/pull/13 ?

**Comment by hoatle** [comment]
hello guys, rename done:

pypi: https://pypi.python.org/pypi/Flask-Classful
docs: http://flask-classful.teracy.org/

**Comment by svenstaro** [comment]
Good job!

**Comment by bivald** [comment]
Awesome, when any of us get some time we could also submit it to  http://flask.pocoo.org/extensions/ http://flask-menu.readthedocs.org/en/latest/
